matt-rich-anderson/property-goat
# Ma notes implementation

#### Changes Made
1. Modified file `NoteForm.js` to include a form that allows you to add a note for a chosen property.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout ma-notesImplementation
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    >From Dashboard click Add Note button. Choose property, enter message, click 'Priority' and submit. You should be returned to Dashboard, where you should find the property you added the note to and click on it to see details, You should see an added note, highlighted in red.
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
